### ISSUE TITLE
If you use the mouse to click on a task, keyboard navigation breaks

### ISSUE THREAD

**Issue by nd-smith** [nd-smith/taskui-python#5]
## Bug Description
If you click on a task with your mouse, the column is removed from focus and keyboard nav doesn't work

## Steps to Reproduce
1. Launch the app
2. Click a task in column 1
3. Notice 'active' column style disappears
4. Hit arrow up arrow down
5. Notice task selector doesn't move

## Expected Behavior
Clicking a task with the mouse should not remove the currently active column

## Actual Behavior
Clicking a task with the mouse causes active column state to be lost
